src/plays/product-entry-form/ProductEntryForm.css

```diff
@@ -0,0 +1,166 @@
+:root {
+	--themeColor: #576e85;
+}
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+body{
```

Comment:
Don;t override selectors directly. It will change the style of other places.

Reply:
alright, let me remove the background color for now.